python/docs/jsonschema_tutorial.rst

```diff
@@ -0,0 +1,161 @@
+    :start-after: # tutorial-point-layout-start
+    :end-before: # tutorial-point-layout-end
+
+And pack the points using the Python built-in ``struct`` and ``base64`` libraries.
+
+.. literalinclude:: ../examples/jsonschema/pointcloud_csv_to_mcap.py
+    :start-after: # tutorial-pack-points-start
+    :end-before: # tutorial-pack-points-end
+
+To set the pointcloud timestamp, the ``foxglove.PointCloud`` schema expects a pair of seconds
+and nanoseconds:
+
+.. literalinclude:: ../examples/jsonschema/pointcloud_csv_to_mcap.py
+    :start-after: # tutorial-timestamp-start
+    :end-before: # tutorial-timestamp-end
+
+And finally we set an arbitrary pose and ``frame_id``.
+
+.. literalinclude:: ../examples/jsonschema/pointcloud_csv_to_mcap.py
+    :start-after: # tutorial-pose-frame-id-start
+    :end-before: # tutorial-pose-frame-id-end
+
+Writing the MCAP
+----------------
+
+Now that the pointcloud is built, we can write it into an MCAP. We'll start with some imports from
+the python MCAP library:
+
+.. literalinclude:: ../examples/jsonschema/pointcloud_csv_to_mcap.py
+    :start-after: # tutorial-mcap-imports-start
+    :end-before: # tutorial-mcap-imports-end
+
+Then we'll open a file for writing and write a header. Note that we can chose whatever name we
+want for both ``profile`` and ``library``, but the profile must start with ``x-``.
+
+.. literalinclude:: ../examples/jsonschema/pointcloud_csv_to_mcap.py
+    :start-after: # tutorial-write-header-start
+    :end-before: # tutorial-write-header-end
+
+After writing a header, we can create a "channel" of messages to contain our pointcloud. The schema
+name and content informs Foxglove Studio that it can parse and display this message as a pointcloud.
+
+.. literalinclude:: ../examples/jsonschema/pointcloud_csv_to_mcap.py
+    :start-after: # tutorial-write-channel-start
+    :end-before: # tutorial-write-channel-end
+
+Finally, we can write our message and ``finish()`` the MCAP writer. ``finish()`` writes the summary
+and footer to the MCAP file.
+
+.. literalinclude:: ../examples/jsonschema/pointcloud_csv_to_mcap.py
+    :start-after: # tutorial-write-message-start
+    :end-before: # tutorial-write-message-end
+
+That's it! Once this is done, we have a valid MCAP file containing a pointcloud message. We can
+check this with the MCAP CLI tool:
+
+.. code-block:: bash
+
+    $ mcap info output.mcap 
+    library: my-excellent-library
+    profile: x-jsonschema
+    messages: 1
+    duration: 0s
+    start: 2011-11-04T01:32:10.881030912+11:00 (1320330730.881030912)
+    end: 2011-11-04T01:32:10.881030912+11:00 (1320330730.881030912)
+    compression:
+            zstd: [1/1 chunks] (40.38%) 
+    channels:
+            (1) /pointcloud  1 msgs (+Inf Hz)   : foxglove.PointCloud [jsonschema]  
+    attachments: 0
+    $ mcap doctor output.mcap
+    Examining output.mcap
+
+We can view the result in Foxglove Studio by navigating to `<https://studio.foxglove.dev>`_
```

Comment:
```suggestion
View your point cloud in Foxglove Studio by navigating to `<https://studio.foxglove.dev>`_
```

Reply:
Must mention adding a 3D panel, as that visualization will not automatically add itself.